eslint-plugin-expensify/CONST.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
         PREFER_IMPORT_MODULE_CONTENTS: 'Do not import individual exports from local modules. Prefer \'import * as\' syntax.',
         PREFER_ONYX_CONNECT_IN_LIBS: 'Only call Onyx.connect() from inside a /src/libs/** file. React components and non-library code should not use Onyx.connect()',
         PREFER_UNDERSCORE_METHOD: 'Prefer \'_.{{method}}\' over the native function.',
+        PREFER_CONSTRUCTORS_WITH_PROPS: 'Constructor should accept props.',
```

Comment:
This should be more specific I think..
```suggestion
        PREFER_REACT_CLASS_CONSTRUCTORS_WITH_PROPS: 'React class constructors must accept props.',
```